### ISSUE TITLE
Docs v1.0.0: sizing-helpers page missing

### ISSUE THREAD

**Issue by jpet710122** [jgthms/bulma#3726]
This is about **Bulma | the Docs**.

### Overview of the problem

This is about the Bulma **Docs** 
I'm using Bulma **version** [1.0.0]

### Description

Clicking on any of the links to **Sizing** under the CSS Helpers part of the contents goes to the **Spacing** page.
There is no `sizing-helpers.html` file in docs/documentation/helpers in the **bulma** repository.

### Steps to Reproduce

1. Go to Bulma docs
2. Click on **CSS Helpers > Sizing** in the contents
3. **Spacing helpers** page is displayed
4. Clicking on **Sizing** button at bottom right of the **Spacing helpers** page reloads **Spacing helpers**.

### Expected behavior

Links to **Sizing helpers** should display the Sizing helpers page.

### Actual behavior
**Spacing helpers** page is displayed instead.

**Comment by jgthms** [comment]
Good point. I believe it was changed to the typography helper page. I'll remove it for now.
